_data/security.yml

```
@@ -63,6 +63,13 @@ websites:
       - u2f
     doc: https://support.cloudflare.com/hc/en-us/articles/200167906
 
+  - name: CloudGuard Dome9
+    url: https://www.checkpoint.com/dome9/
+    img: cloudguard.png
+    tfa:
+      - totp
+    doc: https://supportcenter.checkpoint.com/supportcenter/portal?solutionid=sk144515
```

I only seem to be able to access the doc page with the `eventSubmit_doGoviewsolutiondetails=` parameter in the url. The current doc link just takes me to the support homepage

```suggestion
    doc: https://supportcenter.checkpoint.com/supportcenter/portal?eventSubmit_doGoviewsolutiondetails=&solutionid=sk144515
```